tests/cli/functional/test_worklock_cli_functionality.py

```diff
@@ -364,16 +353,17 @@ def test_participant_status(click_runner,
     result = click_runner.invoke(worklock, command, catch_exceptions=False)
     assert result.exit_code == 0
     
-    expected_calls = ('check_claim',
-                      'eth_to_tokens',
-                      'get_deposited_eth',
-                      'get_eth_supply',
-                      'get_base_deposit_rate',
-                      'get_bonus_lot_value',
-                      'get_bonus_deposit_rate',
-                      'get_bonus_refund_rate',
-                      'get_base_refund_rate',
-                      'get_completed_work',
-                      'get_refunded_work')
+    expected_calls = (mock_worklock_agent.check_claim,
+                      mock_worklock_agent.eth_to_tokens,
+                      mock_worklock_agent.get_deposited_eth,
+                      mock_worklock_agent.get_eth_supply,
+                      mock_worklock_agent.get_base_deposit_rate,
+                      mock_worklock_agent.get_bonus_lot_value,
+                      mock_worklock_agent.get_bonus_deposit_rate,
+                      mock_worklock_agent.get_bonus_refund_rate,
+                      mock_worklock_agent.get_base_refund_rate,
+                      # 'get_completed_work',  # TODO Yes or no?
```

Comment:
??? - what's going on here? 👀

Reply:
Erhm, Needs a close eye to determine if this is an expected call or not - I was seeing some flaky test behavior with this line uncommented.

Reply:
Does this need an issue?